assets/i18n/fr.po

```diff
@@ -6793,39 +6834,60 @@ msgid ""
 "yourself using multiple devices or want to share an in-progress campaign with friends so "
 "everyone can collaborate on it, this is the feature for you."
 msgstr ""
+" Enfin, la possibilité de partager des campagnes est sortie sur iOS et Android ! C'était "
+"l'une des fonctionnalités les plus demandées depuis le début, mais comme elle a nécessité "
+"l'écriture (et le paiement) d'un ensemble de serveurs complètement nouveau, il m'a fallu "
+"un certain temps pour y parvenir. Vous pouvez bien sûr continuer à utiliser l'application "
+"exactement comme elle est, mais si vous vous retrouvez à utiliser plusieurs appareils ou "
+"si vous voulez partager une campagne en cours avec des amis pour que tout le monde puisse "
+"collaborer, cette fonctionnalité est faite pour vous."
 
 msgctxt "releaseNotes"
 msgid ""
 "- First sign up for an <b>Arkham Cards</b> account via the <i>Settings</i> tab, to "
 "unlock these features. Don't forget to set your handle so your friends can find you on "
 "the app."
 msgstr ""
+"- Commencez par vous créer un compte <b>Arkham Cards</b> via l'onglet <i>Paramètres</i>, "
+"pour débloquer ces fonctionnalités. N'oubliez pas de définir votre identifiant pour que "
+"vos amis puissent vous retrouver sur l'application."
 
 msgctxt "releaseNotes"
 msgid ""
 "- On the main screen of every campaign (or when creating a new one), there is a new "
 "option to 'Upload' the campaign to the cloud. This will cause the campaign to be synced "
 "between all of your devices."
 msgstr ""
+"- Sur l'écran principal de chaque campagne (ou lors de la création d'une nouvelle "
+"campagne), il y a une nouvelle option pour " Charger " la campagne sur le cloud. Cela "
```

Comment:
When a quote mark is used inside a line, make sure to 'escape' it using a \ character.  
i.e.: 
```
pour \" Charger \" la campagne
```